torch/utils/data/datasets/common.py

```diff
@@ -2,6 +2,41 @@
 import fnmatch
 import warnings
 from typing import List, Union, Iterable
+from io import BufferedIOBase
+
+class StreamWrapper:
```

Comment:
Need to be Child of Stream class or similar, to make sure that isinstance work correctly.

Reply:
Double checked the BufferedIOBase which is used as streaming handle. It inherited from IOBase which has __del__ implemented calling close() already. I'll remove this StreamWrapper, no longer needed.